src/posts/2022-11-27-define-once/define-once.md

```diff
@@ -0,0 +1,355 @@
+---
+title: Define a python function once and for all
+subtitle: A terrible hack that ends up being surprisingly useful
+---
+
+Sometimes when we have a function that takes a long time to run, and we would
+really rather avoid running the function more often than absolutely necessary.
```

Comment:
This sentence seems grammatically off. Perhaps remove "when"?

Reply:
Done!